includes/admin/class-wcmp-export-consignments.php

```diff
@@ -210,17 +210,22 @@ public function setCustomItems(): void
      */
     private function getTotalWeight(): int
     {
-        $packageType       = $this->orderSettings->getPackageType();
-        $emptyParcelWeight = (int) $this->getSetting(WCMYPA_Settings::SETTING_EMPTY_PARCEL_WEIGHT);
-
-        if (AbstractConsignment::PACKAGE_TYPE_DIGITAL_STAMP === $packageType) {
-            $extraOptions = WCX_Order::get_meta($this->order, WCMYPA_Admin::META_SHIPMENT_OPTIONS_EXTRA);
-            $weight       = $extraOptions['weight'];
-        } else {
-            $weight = $this->order->get_meta(WCMYPA_Admin::META_ORDER_WEIGHT);
+        $weight = $this->order->get_meta(WCMYPA_Admin::META_ORDER_WEIGHT);
+
+        switch ($this->orderSettings->getPackageType()) {
+            case AbstractConsignment::PACKAGE_TYPE_PACKAGE:
+                $extraOptions = WCX_Order::get_meta($this->order, WCMYPA_Admin::META_SHIPMENT_OPTIONS_EXTRA);
+
+                $weight = $extraOptions['weight'];
+                break;
+            case AbstractConsignment::PACKAGE_TYPE_DIGITAL_STAMP:
+                $emptyParcelWeight = (int) $this->getSetting(WCMYPA_Settings::SETTING_EMPTY_PARCEL_WEIGHT);
+
+                $weight += $emptyParcelWeight;
+                break;
```

Comment:
zou hier ook niet een case moeten komen voor BBP?

Reply:
Wat moet er anders als het BBP is dan?